### PR TITLE
feat: Impl multi-sig address and transaction

### PR DESCRIPTION
**Pull request by duanyytop**

### CODE REVIEW

**Comment by duanyytop** [comment]
This pr will be released in v0.24.0 and it will be used by Huobi exchange. So please review it as soon as quickly, thanks.

File: console/src/main/java/org/nervos/ckb/example/transaction/CollectUtils.java [hunk 1]
```diff
@@ -51,17 +58,28 @@ public CollectUtils(Api api) {
...
+
+  public List<CellOutput> generateOutputs(
+      List<Receiver> receivers, String changeAddress, BigInteger fee, CodeHashType codeHashType)
+      throws IOException {
     if (fee.compareTo(BigInteger.ZERO) < 0) {
       throw new IOException("Transaction fee should not be smaller than zero");
     }
     List<CellOutput> cellOutputs = new ArrayList<>();
-    AddressUtils addressUtils = new AddressUtils(Network.TESTNET);
+    AddressUtils addressUtils = new AddressUtils(Network.TESTNET, codeHashType);
```
Comment (xxuejie) [inline console/src/main/java/org/nervos/ckb/example/transaction/CollectUtils.java]: I bet Huobi will want to use mainnet addresses as well, should we make this tunable?

Comment (duanyytop) [reply console/src/main/java/org/nervos/ckb/example/transaction/CollectUtils.java]: I see. This is just a demo for huobi(in actually they also just need a demo), and using testnet address can be run in ckb testnet blockchain.

Comment (xxuejie) [reply console/src/main/java/org/nervos/ckb/example/transaction/CollectUtils.java]: Okay I found out my mistake: I only look at the file name `CollectUtils.java` but didn't see the `example` part in the package name, I thought this is actual SDK code, in this case never mind.

Comment (duanyytop) [reply console/src/main/java/org/nervos/ckb/example/transaction/CollectUtils.java]: OK. Thanks. I will also make sure mainnet address is ok in my local computer.

Comment (duanyytop) [reply console/src/main/java/org/nervos/ckb/example/transaction/CollectUtils.java]: I tried with mainnet address in my local computer and it did work well.

**Pull request merged**